package.json

```diff
@@ -52,6 +52,7 @@
     "request": "^2.88.2",
     "sinon": "^9.2.4",
     "typescript": "^4.4.3",
+    "watch": "1.0.2",
```

Comment:
Why?

Reply:
This is for theme build watch. At the moment, we have `watch` command in theme build script, but we don't have `watch` package. I'll create a separate PR for this.